services/business/utils.go

```diff
@@ -0,0 +1,18 @@
+package business
+
+import (
+	"k8s.io/api/core/v1"
+
+	"github.com/kiali/kiali/services/models"
+)
+
+func hasIstioSideCar(pods []v1.Pod) bool {
```

Comment:
IMO, I would rather put this method inside models/pod.go. There are all the methods that deals with pod logics.
I guess that it would be great to have a method like `pod.HasIstioSideCar()`